src/rp2_common/hardware_flash/flash.c

```diff
@@ -165,7 +165,7 @@ static void __no_inline_not_in_flash_func(flash_do_cmd)(const uint8_t *txbuf, ui
 // Use standard RUID command to get a unique identifier for the flash (and
 // hence the board)
 
-static_assert(FLASH_UNIQUE_ID_SIZE_BYTES == FLASH_RUID_DATA_BYTES);
+static_assert(FLASH_UNIQUE_ID_SIZE_BYTES == FLASH_RUID_DATA_BYTES, "");
```

Comment:
Might make sense to use the same message as https://github.com/raspberrypi/pico-sdk/blob/develop/src/rp2_common/pico_unique_id/unique_id.c#L10 ?